kurtosis-devnet/pkg/util/docker.go

```diff
@@ -133,3 +176,558 @@ func DestroyDockerResources(ctx context.Context, enclave ...string) error {
+		}
+	}()
+
+	fmt.Printf("Stopping current Traefik container\n")
+	timeout := int(10)
+	if err := apiClient.ContainerStop(ctx, traefikContainer.ID, container.StopOptions{
+		Timeout: &timeout,
+	}); err != nil {
+		return fmt.Errorf("failed to stop container: %w", err)
+	}
+
+	if err := apiClient.ContainerRemove(ctx, traefikContainer.ID, container.RemoveOptions{
+		RemoveVolumes: true,
+		Force:         true,
+	}); err != nil {
+		return fmt.Errorf("failed to remove container: %w", err)
+	}
+
+	newContainer, err := recreateTraefikContainer(ctx, apiClient, containerName, containerImage, portBindings, tempDir, containerInfo.Config.Labels)
+	if err != nil {
+		return fmt.Errorf("failed to recreate container: %w", err)
+	}
+
+	fmt.Printf("Created new Traefik container: %s\n", newContainer.ID[:12])
+	for _, networkName := range networks {
+		if err := apiClient.NetworkConnect(ctx, networkName, newContainer.ID, nil); err != nil {
+			return fmt.Errorf("failed to connect to network %s: %w", networkName, err)
+		}
+	}
+	time.Sleep(3 * time.Second)
+
+	// Check if container is running
+	runningContainers, err := apiClient.ContainerList(ctx, container.ListOptions{
+		Filters: traefikFilters,
+	})
+	if err != nil {
+		return fmt.Errorf("failed to list containers: %w", err)
+	}
+
+	containerRunning := false
+	for _, c := range runningContainers {
+		if c.ID == newContainer.ID {
+			containerRunning = true
+			break
+		}
+	}
+
+	if !containerRunning {
+		logs, _ := apiClient.ContainerLogs(ctx, newContainer.ID, container.LogsOptions{
+			ShowStdout: true,
+			ShowStderr: true,
+		})
+		if logs != nil {
+			logData, _ := io.ReadAll(logs)
+			fmt.Printf("Container logs:\n%s\n", string(logData))
+		}
+		return fmt.Errorf("new Traefik container failed to start")
+	}
+
+	if err := TestRPCEndpoints(ctx, apiClient); err != nil {
+		fmt.Printf("RPC access test failed: %v\n", err)
+	}
+
+	fmt.Printf("Traefik routing fix completed successfully\n")
+
+	return nil
+}
+
+func createTempConfigDir(ctx context.Context, networkID string) (string, error) {
+	tempDir, err := os.MkdirTemp("", "traefik-fix-*")
+	if err != nil {
+		return "", fmt.Errorf("failed to create temp directory: %w", err)
+	}
+
+	apiClient, err := NewDockerClient()
+	if err != nil {
+		os.RemoveAll(tempDir)
+		return "", fmt.Errorf("failed to create Docker client: %w", err)
+	}
+	defer apiClient.Close()
+
+	networks, err := apiClient.NetworkList(ctx, network.ListOptions{})
+	if err != nil {
+		os.RemoveAll(tempDir)
+		return "", fmt.Errorf("failed to list networks: %w", err)
+	}
+
+	var traefikConfig strings.Builder
+	traefikConfig.WriteString(`# Traefik configuration with corrected network ID
+api:
+  dashboard: true
+  insecure: true
+
+entryPoints:
+  web:
+    address: ":9730"
+  traefik:
+    address: ":9731"
+
+providers:
+`)
+
+	for _, net := range networks {
+		// Only include Kurtosis networks (skip bridge, host, none)
+		if !strings.HasPrefix(net.Name, "kt-") {
```

Comment:
I would then revert the condition since you have a known list of the negative cases:

```go
switch net.Name {
 case "bridge", "host", "none":
   continue
}
```